src/app/app.module.ts

```diff
@@ -46,6 +46,9 @@ import { LoaderComponent } from './loader/loader.component';
 		FormsModule,
 		ReactiveFormsModule
 	],
+	schemas: [
```

Comment:
Why do you need the schema?

Reply:
I thought in this place need to use schemas